baton/hub.go

```diff
@@ -40,45 +40,57 @@ func (h hub) run() {
 				close(c.send)
 			}
 		case rawMsg := <-h.receive:
-			fmt.Println(rawMsg)
 			var cmd command
 			if err := json.Unmarshal(rawMsg.data, &cmd); err != nil {
 				fmt.Println("nu")
 				log.Println(err)
 				break
 			}
-
+			fmt.Println(cmd)
```

Comment:
Perhaps make this a log statement instead and add context for the line? Might make sense to use a package like https://github.com/Sirupsen/logrus so we can have logging levels and disable certain levels.

```go
log.Println("Incoming message:", cmd)
```